documentation/modules/ROOT/pages/development/engine.adoc

```diff
@@ -355,6 +355,24 @@ DebeziumEngine<RecordChangeEvent<SourceRecord>> engine = DebeziumEngine.create(C
         .build();
 ----
 
+If JSON or Avro format is used then the code would look like:
+
+[source,java,indent=0]
+----
+
+class MyChangeConsumer implements DebeziumEngine.ChangeConsumer<ChangeEvent<String, String>> {
+  public void handleBatch(List<ChangeEvent<String, String>> records, RecordCommitter<ChangeEvent<String, String>> committer) throws InterruptedException {
```

Comment:
Let's make sure this line doesn't wrap when rendered and wrap ourselves instead.